common/azure-spring-boot-starter-parent/pom.xml

```diff
@@ -225,8 +225,6 @@
                     <instrumentation>
                         <excludes>
                             <exclude>com/microsoft/azure/spring/common/GetHashMac.class</exclude>
```

Comment:
one concern on parent-pom is that if user used parent package as parent, build will break.